URBNSwiftAlert/Classes/AlertViewController.swift

```diff
@@ -284,6 +284,7 @@ extension AlertViewController {
     
     public func addTextfield(configurationHandler: ((UITextField) -> Void)) {
         let tf = UITextField()
+        tf.accessibilityIdentifier = "alertTextField"
```

Comment:
just note if there are multiple text fields, they all will have the same identifier